cmd/cli/get.go

```diff
@@ -80,4 +85,6 @@ func init() {
 	rootCmd.AddCommand(getCmd)
 	getCmd.Flags().StringVar(&getArgs.KubeconfigPath, "kubeconfig", "", "Path to kubeconfig")
 	getCmd.Flags().StringVar(&getArgs.Format, "format", "table", "The format in which to display results (currently only 'table' supported)")
+	getCmd.Flags().StringVar(&upsertArgs.AsUser, "as", "", "Username to impersonate for the operation")
```

Comment:
Wondering if these flags can be rootCmd flags and carry over to all commands as a result

Reply:
I had them initially as root but then they show up in `version -h` if your fine with that more then happy to change it

Reply:
Good point - thats fine, lets leave as is then